packages/orbit-components/src/Tabs/components/Tab/index.tsx

```diff
@@ -31,7 +31,7 @@ const Tab = ({
         "flex flex-1 flex-row items-center justify-between",
         "appearance-none border-0",
         "min-w-fit",
-        "relative box-border",
```

Comment:
This fixes the problem with the outline in the parent element.

I've tried different scenarios and I didn't understand why `position: relative` is needed.

Let me know if you find something breaking 😄

Reply:
I was poking around and came to the same conclusion 🤔